qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -0,0 +1,90 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Base class for calibration-type experiments."""
+
+from abc import abstractmethod
+from typing import Optional
+
+from qiskit.providers.options import Options
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework.base_experiment import BaseExperiment
+from qiskit_experiments.framework.experiment_data import ExperimentData
+
+
+class BaseCalibrationExperiment(BaseExperiment):
```

Comment:
If this is a baseclass this should take ABC too (just to make sure).

Reply:
Done here: 6a4c454